# Drug test
In terms of random variables, we assume that both sample came from different random varibles (X, Y), $\mathop{\mathbb{E}}X = \mu_{cont}$ and $\mathop{\mathbb{E}}Y = \mu_{treat}$. Our null hypothesis is that mean of random variables, from which both samples are get, are the same, and the cure doesn't give any effect.
$$H_0: \mu_{cont} = \mu_{treat}$$
The alternative is that there is an effect that gives the drug, but we don't know in advance, what kind of effect would happen, so the alternative is that:
$$H_1: \mu_{cont} \neq \mu_{treat}$$
This alternative is two-sided, so, to perform statistical test, we have to use 2 sample 2 sided t-test, with **scipy.stats.ttest_ind** and parameter *equal_var=False*

In [1]:
from scipy.stats import ttest_ind

In [3]:
control_sample = [6,7,7,5,7,8,8,7,7,7]
treatment_sample = [7,6,6,5,5,6,7,5,5,8]
ttest_ind(treatment_sample, control_sample, equal_var=False)

Ttest_indResult(statistic=-2.076923076923078, pvalue=0.05290570806269817)

From result of 2-sample t-test, we can say, that with significance level around 6%, we can say that there is an effect, created by our drug. But, from observed t-statistics, which is negative, it is obvious that mean of treatment sample is less, that control sample, so new alternative declared:
$$H_1^{'}:\mu_{cont} > \mu_{treat}$$
This means, that now we expect that our drug is decreasing time for curing, our drug is effective and we can invest to it. The probability to make Type I error in 1-sided t-test (p-value) is: $0.053 / 2 = 0.0265$. If we assume, that our significance level is 5%, we can say that out result is statistically significant and with significance level of 5%, the drug is effective and it decreases time, needed for cure. But, there is a probablity (around 3%) that the drug is ineffective and investment in it cause a Type I error, since data size of such clinical trial is very small and with only 6% of significance level we can say, that there is any effect.